doc/api/buffer.md

```diff
@@ -1597,6 +1597,10 @@ changes:
     pr-url: https://github.com/nodejs/node/pull/18395
     description: Removed `noAssert` and no implicit coercion of the offset
                  to `uint32` anymore.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/24298
+    description: Reworked internal numerical read/write functions to use
```

Comment:
I don't think changes in implementation like this should go into the documentation as it doesn't affect the end user.